### PR TITLE
added KYC 311 pipeline script

### PR DESCRIPTION
**Pull request by prestinomills**
Added the KYC 311 pipeline script to automate weekly

### CODE REVIEW

File: civis/KYC/kyc_311_to_esri.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[12]:
```
Comment (ian-r-rose) [inline civis/KYC/kyc_311_to_esri.py:4]: Can you remove these `In[]` prompts? They don't really add much useful info in this context.

File: civis/KYC/kyc_311_to_esri.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+myla311_layer = '4db3e9c3d13543b6a686098e0603ddcf'
+
+
+# In[14]:
+
+
+#---For 311
+def prep_311_data(file):
+    catalog = intake_civis.open_redshift_catalog()
+    expr = catalog.public.import311.to_ibis()
```
Comment (ian-r-rose) [inline civis/KYC/kyc_311_to_esri.py:32]: Nice

**Comment by tiffanychu90** [comment]
@prestinomills For future reference: https://github.com/CityOfLosAngeles/notebook-demos/blob/master/ibis-query-311.py. You can add more examples there, and we'll probably add more examples in the future too.

File: civis/KYC/kyc_311_to_esri.py (hunk 1 above)
Comment (hunterowens) [reply civis/KYC/kyc_311_to_esri.py:4]: removed.

**Pull request merged**